src/index.html

```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <div class="logo">
+          <img src="./images/logo.png" alt="logo">
+        </div>
+        <nav class="menu">
+          <ul>
+            <div class="active__border"></div>
+            <li class="nav__items">
```

Comment:
It's one item. Class should be named accordingly.
```suggestion
            <li class="nav__item">
```